src/lib/viewers/BaseViewer.js

```diff
@@ -628,7 +628,8 @@ class BaseViewer extends EventEmitter {
      * @return {Object} An object containing configuration properties.
      */
     getViewerAnnotationsConfig() {
-        const config = this.getViewerOption('annotations') || {};
+        const option = this.getViewerOption('annotations');
+        const config = option !== null && option !== undefined ? option : {};
```

Comment:
`const config = (option !== null && option !== undefined) ? option : {};`

but also does the previous version not work? 
`const config = this.getViewerOption('annotations') || {};`